Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1382,9 +1421,11 @@ private extension Purchases {
     }
 
     func dispatchSyncSubscriberAttributes() {
-        operationDispatcher.dispatchOnWorkerThread {
+        #if !ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION
+        self.operationDispatcher.dispatchOnWorkerThread {
```

Comment:
we can probably group this one and the one for `(self as DeprecatedSearchAdsAttribution).postAppleSearchAddsAttributionCollectionIfNeeded()` a few lines above into one, since this method is only called from there, right?

Reply:
This is also called from `applicationWillResignActive`.